desktop/windows/release-notes/index.md

```diff
@@ -23,15 +23,42 @@ This page contains information about the new features, improvements, known issue
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
-## Docker Desktop 4.4.4
-2022-01-24
+## Docker Desktop 4.5.0
+2022-02-10
 
 > Download Docker Desktop
 >
 > [For
 > Windows](https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64){:
 > .button .primary-btn }
 
+### New
+
+- Add Desktop Version to 'docker version' output.
+
+### Upgrades
+
+- Bump ECR Credential helper to 0.6.0
```

Comment:
```suggestion
- Bump ECR Credential helper to 0.6.0
```
We need a link. Also the wording is quite constrained, see the previous release notes.